test/app/gateways/default.gateway.ts

```diff
@@ -0,0 +1,11 @@
+import { WebSocketGateway, SubscribeMessage } from '@nestjs/websockets';
+import { AppService } from '../app.service';
+
+@WebSocketGateway({ path: 'default' })
```

Comment:
Can we exclude the websocket tests as well in this release? *If* these break then it should'nt be a thing for this release since we don't support WS anyway. This means we also have to exclude other modules we have inside `test/app`. For now we only need the `ControllersModule`.

Reply:
The test code won't be released anyways, it'll only be in the GitHub repo, so I'm not too worried about this. The only thing that should be in the release is the `dist`